content/Parents_content.php

```diff
@@ -37,3 +37,20 @@
 		</p>
 	</div>
 </div>	
+<div class="post">
+	<h1 class="title"><a>New Year Student Signups</a></h1>
+		</p>
+	</div>
+</div>
+<div class="post">
+	<h1 class="title"><a>Want to join for our team?</a></h1>
+	<p class="byline"><small>Posted on August 24th, 2015</small></p>
+	<div class="entry">
+		<p>
+Please fill out the below papers and bring them with you to the next open shop night: </br>
```

Comment:
it is a 👃 to use `<br>` also the the slash is not needed for html5 (and it is on the wrong side anyway)
it would be better to wrap the text node in the `<p>` block, and just throw your `<ul>` block below it. 
You NEVER should leave text nodes as peers to html elements. The render cycle has a super bad time when flowing the document.

I would suggest a structure like:
```html
<div class="entry">
  <p>
    Please fill out the below papers and bring them with you to the next open shop night:
  </p>
  <ul class="papers">
    <li class="papers__item"><a target="_blank" href="some-url">Some Text</a></li>
    <li class="papers__item"><a target="_blank" href="some-url">Some Text</a></li>
    <li class="papers__item"><a target="_blank" href="some-url">Some Text</a></li>
  </ul>
</div>
```
Structuring your html this way is good for the soul AND the flow cycle.